src/instructlab/cli/data/generate.py

```diff
@@ -168,16 +168,20 @@
 @click.option(
     "--student-model-id",
     # help="ID of the custom model you want to use as the student model.",
-    default=None,
     type=click.STRING,
-    show_default=True,
+    show_default=False,
+    config_class="general",
+    config_sections="student_model_id",
+    cls=clickext.ConfigOption,
 )
 @click.option(
     "--teacher-model-id",
     # help="ID of the custom model you want to use as the teacher model.",
-    default=None,
     type=click.STRING,
-    show_default=True,
+    show_default=False,
+    config_class="general",
```

Comment:
idk if you need config_sections if its in the top level of `general`

Reply:
hmmm, doesnt seem to be making a difference either way